### PR TITLE
Remove broken boards

### PR DESCRIPTION
**Pull request by rminnich**
These are done one commit per board so reversion is easy and boards can be brought back.

### CODE REVIEW

**Comment by rminnich** [comment]
where are those board builds coming from? What script? I have removed them from CI!
This setup is too complex.

**Comment by rminnich** [comment]
I confess I'm lost on just how it picks what targets to try to build. But we're ready to make this action required.

**Comment by rminnich** [comment]
getting rid of totally broken stuff, and adding it back later, bit by bit, will be easier than fixing everything. We know this because waiting 3 years for people to fix everything has not worked.

I would greatly appreciate a review, but this is going in in a few hours no matter what. This must get fixed.

I would also appreciate anyone who can review the branch protection rules I set up -- I am sure I did some things wrong.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -32,32 +32,11 @@ jobs:
       matrix:
         # The `${{ matrix.mainboard }}` variable is replaced with each of these.
         mainboard:
-          # '-' is used instead of '/' due to restrictions on artifact upload.
-          - aeeon-up
```
Comment (orangecms) [inline .github/workflows/ci.yml:36]: This is the one I had fixed in #95

File: mainboards/Makefile [hunk 2]
```diff
@@ -20,73 +20,8 @@ docker-build: Dockerfile
 
 docker-run-aeeon-up: docker-build
 	docker run --rm -v "$(shell pwd)/aeeon/up":/$(CONTAINER_NAME) $(CONTAINER_NAME) 'make fetch flashkernel'
-
```
Comment (orangecms) [inline mainboards/Makefile:23]: let's remove this file entirely, it looks like it just creates lots of complexity - I hadn't even known about it

File: .github/workflows/ci.yml (hunk 1 above)
Comment (rminnich) [reply .github/workflows/ci.yml:36]: done. I wanted to bring them back one commit at a time, it makes bisection easy, as well as reversion.

File: mainboards/Makefile (hunk 2 above)
Comment (rminnich) [reply mainboards/Makefile:23]: looking at that Dockerfile, it *does* allow you to do some amount of testing in mainboards/Makefile.

If you are sure we can remove it, no problem though. Not sure the test is even meaningful at this point.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (rminnich) [reply .github/workflows/ci.yml:36]: no, it's still broken, there is no key.pub. Should we generate it each time? Or just touch it?

Comment (rminnich) [reply .github/workflows/ci.yml:36]: nvm got it. We should probably not try to build cpuuroot in CI (yet) until we resolve the key thing.

File: mainboards/Makefile (hunk 2 above)
Comment (rminnich) [reply mainboards/Makefile:23]: since this breaks nothing, can we leave it here and address it in another PR?

**Comment by rminnich** [comment]
This small number of machines gives us a base from which to build a better makefile setup. Also, it's multi-arch, thanks to ampere.

File: mainboards/Makefile (hunk 2 above)
Comment (ChiaoY) [reply mainboards/Makefile:23]: The docker support makes the Makefile complex as @orangecms said. We should have better approach if we want to reserve the docker support.

File: mainboards/Makefile [hunk 3]
```diff
@@ -20,73 +20,8 @@ docker-build: Dockerfile
 
 docker-run-aeeon-up: docker-build
```
Comment (ChiaoY) [inline mainboards/Makefile]: If we still want docker build support for few boards, shouldn't we keep the Dockerfile?

**Comment by pamolloy** [comment]
Hi @rminnich, could you reword the commit messages to reference the board? For example, something like `marvel/mcchiatobin: prune broken board`?

**Comment by orangecms** [comment]
I'm personally okay with the commit messages as they are, would just like to see a graveyard.md listing which commit removed which board, similar to https://github.com/oreboot/oreboot/blob/main/graveyard.md

**Review by orangecms — APPROVED** [review]

**Comment by orangecms** [comment]
graveyard.md would be for another PR once this here is merged, so that the final commit hashes are known, so let's do this here

**Pull request merged**